tests/test_example_filetype_format.py

```diff
@@ -8,9 +8,38 @@
 
 @pytest.fixture
 def filetype_format_class(syn):
+    yield FileTypeFormat(syn, "SAGE", ancillary_files=[["mocked"]])
+
+
+@pytest.fixture
+def filetype_format_class_no_ancillary(syn):
     yield FileTypeFormat(syn, "SAGE")
 
 
+def test_that_validate_no_cross_file_without_ancillary(
```

Comment:
Would add more tests here if we update what we are checking for in the `ancillary_files`